hack/build/cluster-sync-test.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
I think we would be better off not committing this file. It might discourage anyone who wants to make further changes to the sync code by needing to understand what it does.